src/pages/sol/material_properties.astro

```diff
@@ -44,7 +38,7 @@ Loading in this region results <strong>elastic behavior</strong>, meaning the ma
 
 
  Hooke's law is used for small deformations in the elastic region. The Young's modulus is the slope <InlineEquation equation="E" />.
-<DisplayEquation equation="\\sigma = E\\varepsilon\\" />
+<DisplayEquation equation="\\sigma = E\\varepsilon\\" title="Hooke's law:" background="True" />
```

Comment:
We are using sentence form for titles (ending in a period, not a colon). There are more of these in this PR, not just this one.
```suggestion
<DisplayEquation equation="\\sigma = E\\varepsilon\\" title="Hooke's law." background="True" />
```

Reply:
Fixed!